src/main/resources/assets/jei/lang/de_de.lang

```diff
@@ -49,60 +49,60 @@ config.jei.title=%MODNAME-Konfiguration
+config.jei.interface.overlayEnabled=Aktiviere die Items-Liste
+config.jei.interface.overlayEnabled.comment=Zeige die Liste der Items neben einer geöffnete Benutzer-Oberfläche.
 
 config.jei.search=Suche
 config.jei.search.comment=Suchleisteneinstellungen
 config.jei.search.modNameSearchMode=@ModName
-config.jei.search.modNameSearchMode.comment=Suchmodus für Mod-Namen (Präfix: @)
+config.jei.search.modNameSearchMode.comment=Suchmodus für den Mod-Namen (Präfix: @)
 config.jei.search.tooltipSearchMode=#Tooltip
-config.jei.search.tooltipSearchMode.comment=Suche für Tooltips (Präfix: #)
+config.jei.search.tooltipSearchMode.comment=Suche für den Tooltip (Präfix: #)
 config.jei.search.oreDictSearchMode=$OreDictionary
-config.jei.search.oreDictSearchMode.comment=Suchmodus für Begriffe im Ore Dictionary (Präfix: $)
+config.jei.search.oreDictSearchMode.comment=Suchmodus für Begriffe im Ore Dictionary(Erze-Verzeichnis) (Präfix: $)
 config.jei.search.creativeTabSearchMode=%%Kategorie
-config.jei.search.creativeTabSearchMode.comment=Suchmodus für Kategorien im Kreativmodus-Inventar (Präfix: %)
+config.jei.search.creativeTabSearchMode.comment=Suchmodus für die Kategorien im Kreativmodus-Inventar (Präfix: %)
 config.jei.search.colorSearchMode=^Farbe
-config.jei.search.colorSearchMode.comment=Suchmodus für Item-Farben (Präfix: ^)
+config.jei.search.colorSearchMode.comment=Suchmodus für die Items-Farben (Präfix: ^)
 config.jei.search.resourceIdSearchMode=&ResourceId
-config.jei.search.resourceIdSearchMode.comment=Search mode for resources ids (prefix: &)
+config.jei.search.resourceIdSearchMode.comment=Suchmodus für die "Resource-Id" (prefix: &)
 
 config.jei.advanced=Erweitert
 config.jei.advanced.comment=Erweiterte Konfigurationsoptionen, die ändern, wie JEI funktioniert.
 config.jei.advanced.itemBlacklist=Aktiviere Blacklist
-config.jei.advanced.itemBlacklist.comment=Gegenstände, die nicht angezeigt werden sollen. Format: modId[:name[:meta]]. Der Listeneditiermodus entfernt und fügt automatisch neue Gegenstände zu dieser Liste hinzu.
-config.jei.advanced.colorSearchEnabled=Aktiviere Farbsuche
-config.jei.advanced.colorSearchEnabled.comment=Suche nach Gegenständen mittels ihrer Farbe und zeige die suchbaren Farben in Tooltips in der Item-Liste.
-config.jei.advanced.debugModeEnabled=Aktiviere Debug-Modus
-config.jei.advanced.debugModeEnabled.comment=Fügt tausende Test-Items und einige Debug-Rezepte hinzu. Nur für Entwickler bestimmt.
+config.jei.advanced.itemBlacklist.comment=Items, die nicht angezeigt werden sollen. Format: modId[:name[:meta]]. Der Listeneditiermodus entfernt und fügt automatisch neue Items zu dieser Liste hinzu.
+config.jei.advanced.colorSearchEnabled=Aktiviere die Farbsuche
+config.jei.advanced.colorSearchEnabled.comment=Suche nach Items mittels ihrer Farbe und zeige die suchbaren Farben in der Schnellinfo(Tooltip) in der Items-Liste.
+config.jei.advanced.debugModeEnabled=Aktiviere den Fehlersuche-Modus
```

Comment:
Uh, what? No, this is not what it means.
```suggestion
config.jei.advanced.debugModeEnabled=Debug-Modus aktivieren
```

Reply:
Hmm... I didn't know it...